source/_integrations/reolink.markdown

```diff
@@ -621,6 +621,58 @@ Prerequisites:
 
 - Turn on (outdoor) lights near the camera to improve image clarity at night once the camera detects a person, vehicle, or animal.
 - Turn off notifications and recording when you get home (based on, e.g., geofencing) and turn it back on when you leave home.
+- Auto pause rich notifications for x time
+
+{% details "Auto pause rich notifications tutorial" icon="mdi:cursor-hand" %}
+
+This tutorial will give you a drop down on your dashboard with different time choices to pause your notifications. When the time is up, the notifications will become active again. It will look like this:
+
+![Overview of end result](/images/integrations/reolink/auto_pause__overview.png)
+
+First create the dropdown from **Settings** > **Devices & services** > **Helpers** > **+ Create Helper** > **Dropdown**. Decide how many time delay choices you want. Add them all to the dropdown like below. Your first entry needs to be "Notifications active" (or simular phrasing) for when the notifications are turned on. The time amounts can be any amount and you can have as many as you want to code.
+
+![Dropdown](/images/integrations/reolink/auto_pause__dropdown.png)
+
+Next, also from the **Helpers** menu create a **Timer**. Leave the time duration all zeros. Click on Restore state and time box.
+
+![Timer](/images/integrations/reolink/auto_pause__timer.png)
+
+Now you will create a new automation script. For the **When** select **+ Add Trigger** > **Entity** > **State** and choose your dropdown box for the entity and in the **From** choose your "Notifications active" or whatever you chose for the top item.
+
+![Automation When dropdown](/images/integrations/reolink/auto_pause__automation_when.png)
+
+Add another trigger using **+ Add Trigger** > **Entity** > **State** and choose your timer for the entity and in the **To** choose "Idle". Now click on the overflow menu (3 dots) of this tigger and press **Edit ID**. In the **Trigger ID** type "TIMER DONE".
+
+![Automation When timer](/images/integrations/reolink/auto_pause__automation_when_timer.png)
+
+There is nothing in the **And if** section. For the **Then do** section choose **add building block** and use **Choose**. You will have as many options as you have times in your dropdown box plus one to reset the dropdown box. First we make the option to reset the dropdown box, this needs to be the first option. Under **Option 1** click **+ Add Condition** > **Other conditions** > **Triggered by**. Now check the box in front of "TIMER DONE".
+
+![Automation Triggered by](/images/integrations/reolink/auto_pause__automation_triggered_by.png)
+
+Add an action under this **Option 1**, choose **Select** as your action and then choose **First**. Then enter your dropdown box as the entity. This will change your dropdown box back to the first item when the timer is done. This will allow your notification automation to run again.
```

Comment:
```suggestion
6. Add an action under this **Option 1**, choose **Select** as your action and then choose **First**. 
   - Then enter your dropdown box as the entity. 
   - This will change your dropdown box back to the first item when the timer is done. 
   - This will allow your notification automation to run again.
```